packages/analytics/analytics-metric-provider/src/components/metricsProviderUtil.ts

```diff
@@ -39,6 +39,7 @@ interface FetcherOptions {
   queryFn: AnalyticsBridge['queryFn']
   averageLatencies: Ref<boolean>,
   abortController?: AbortController
+  refreshCounter?: Ref<number>
```

Comment:
Since the prop has a default value, this option will always be available.
```suggestion
  refreshCounter: Ref<number>
```